api/storage_service.cc

```diff
@@ -403,6 +403,19 @@ void set_storage_service(http_context& ctx, routes& r) {
         });
     });
 
+    ss::repair_async_status_until_finished.set(r, [&ctx](std::unique_ptr<request> req) {
+        return repair_get_status_until_finished(ctx.db, boost::lexical_cast<int>(req->get_query_param("id")))
+                .then_wrapped([] (future<repair_status>&& fut) {
+            ss::ns_repair_async_status::return_type_wrapper res;
+            try {
+                res = fut.get0();
+            } catch(std::runtime_error& e) {
```

Comment:
Space after `catch`.

Reply:
Why just runtime_error?

Reply:
It is a copied from the old ss::repair_async_status.set function. I will fix it.